minimum-cluster-setup.md

```diff
@@ -246,7 +246,7 @@ Read [this tutorial](http://www.tutorialspoint.com/hadoop/hadoop_multi_node_clus
 - Change the `/usr/local/elasticsearch/config/elasticsearch.yml` file as shown below. This is minimal and allows all hosts to act as backup masters in case the acting master goes down. Also all hosts are data/index nodes so can respond to queries and host shards of the index.
 
   ```
-cluster.name: your-app-name
+cluster.name: some-cluster # remember to put the same name in pio-env.sh below
```

Comment:
actually the name of the cluster does not have to be the appname. Many people use the same cluster for multiple models, which implies many appnames so this should be a name to identify the physical cluster of Elasticsearch machines, not an app, which is tied to a dataset in the EventServer.